### PR TITLE
Update duckduckgo.po

### PR DESCRIPTION
**Pull request by mozillakab**
Fully translated with some corrections

### CODE REVIEW

**Comment by mozillakab** [comment]
@SelyanKab @BoFFire 
Please, use this last file to correct if you find any error.

**Comment by BoFFire** [comment]
@mozillakab Thank you <3!

File: locales/kab_DZ/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -3304,12 +3301,12 @@ msgstr "Tabniqt n unadi"
...
+msgstr "Agilal n tqeffalt n unadi"
 
 #. search for a place within the map area currently shown in expanded map
 msgctxt "vertical_map"
 msgid "Search Current Map Area"
-msgstr ""
+msgstr "Anadi deg temnaḍt tamirant"
 
 msgid "Search DuckDuckGo"
 msgstr "Nadidi DuckDuckGo"
```
Comment (BoFFire) [inline locales/kab_DZ/LC_MESSAGES/duckduckgo.po]: "Nadidi DuckDuckGo" should be "Nadi di DuckDuckGo" ?

Comment (BoFFire) [inline locales/kab_DZ/LC_MESSAGES/duckduckgo.po]: "Nadidi DuckDuckGo" should be "Nadi di DuckDuckGo" ?

Comment (mozillakab) [reply locales/kab_DZ/LC_MESSAGES/duckduckgo.po]: "Nadi di DuckDuckGo"

**Comment by BoFFire** [comment]
Ok good :) Thanks ! @mozillakab

File: locales/kab_DZ/LC_MESSAGES/duckduckgo.po (hunk 1 above)
Comment (BoFFire) [reply locales/kab_DZ/LC_MESSAGES/duckduckgo.po]: Tanemmirt !

**Comment by BoFFire** [comment]
Ok, ready to check and merge @jbarrett :)

**Comment by mozillakab** [comment]
Is there someone to merge? We are still waiting to get the complete kab version deployed online.
Thanks

**Comment by jbarrett** [comment]
Thank you @mozillakab and @BoFFire! Really sorry about the delay.

:+1:

**Pull request merged**